.github/schema/schema.yml

```diff
@@ -38,6 +38,31 @@ mapping:
         unique: true
         required: true
 
+  ExpectedVersionInformation:
+    type: seq
+    required: false
+    sequence:
+      - type: map
+        func: check_executables
+        mapping:
+          OriginalName:
+            type: str
+            required: false
+            unique: true
+            pattern: '^[ -~]+$'
```

Comment:
Is this asserting the provided value must be in the ASCII range?

Reply:
yes - this is regex for entries with only 1 or more printable characters.